src/api/libcellml/variable.h

```diff
@@ -205,22 +205,25 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
      *
      * @return @c true if the @p equivalentVariable is in this variable's equivalent variables
      * and @c false otherwise.
+     *
+     * @warning This method is currently needed by our Printer and Validator classes, but this is
+     * not a method that a libCellML user should ever need. It will therefore be removed at some
+     * point.
```

Comment:
I don't think this belongs in the API documentation.  I would prefer it if we marked the method as deprecated instead.

Reply:
Good point, I hadn't thought of that special command. Consider it done.